README.md

```diff
@@ -1,6 +1,91 @@
-# Podman Ansible Module
 
+Ansible Module for Podman
+=========================
+This module will allow to use Podman to manage the images.
```

Comment:
This module allows Ansible to use Podman to manage container images.

Reply:
Updated the README.md with the suggested changes. Thank you.